src/libraries/Fuzzing/DotnetFuzzing/Fuzzers/Base64UrlFuzzer.cs

```diff
@@ -0,0 +1,133 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Buffers;
+using System.Buffers.Text;
+using System.Formats.Asn1;
+using System.Reflection;
+
+namespace DotnetFuzzing.Fuzzers
+{
+    internal sealed class Base64UrlFuzzer : IFuzzer
+    {
+        public string[] TargetAssemblies => [];
+
+        public string[] TargetCoreLibPrefixes => ["System.Buffers.Text"];
+
+        public void FuzzTarget(ReadOnlySpan<byte> bytes)
+        {
+            using PooledBoundedMemory<byte> inputPoisoned = PooledBoundedMemory<byte>.Rent(bytes, PoisonPagePlacement.After);
+            Span<byte> input = inputPoisoned.Span;
+            int encodedLength = Base64Url.GetEncodedLength(bytes.Length);
+            using PooledBoundedMemory<char> destPoisoned = PooledBoundedMemory<char>.Rent(encodedLength, PoisonPagePlacement.After);
+            Span<char> encoderDest = destPoisoned.Span;
+            int maxDecodedLength = Base64Url.GetMaxDecodedLength(encodedLength);
+            using PooledBoundedMemory<byte> decoderDestPoisoned = PooledBoundedMemory<byte>.Rent(maxDecodedLength, PoisonPagePlacement.After);
+            Span<byte> decoderDest = decoderDestPoisoned.Span;
```

Comment:
Nit: this is pretty dense and hard to read... maybe:
```suggestion
            using PooledBoundedMemory<byte> inputPoisoned = PooledBoundedMemory<byte>.Rent(bytes, PoisonPagePlacement.After);
            int encodedLength = Base64Url.GetEncodedLength(bytes.Length);
            int maxDecodedLength = Base64Url.GetMaxDecodedLength(encodedLength);

            using PooledBoundedMemory<char> destPoisoned = PooledBoundedMemory<char>.Rent(encodedLength, PoisonPagePlacement.After);
            using PooledBoundedMemory<byte> decoderDestPoisoned = PooledBoundedMemory<byte>.Rent(maxDecodedLength, PoisonPagePlacement.After);

            Span<byte> input = inputPoisoned.Span;
            Span<char> encoderDest = destPoisoned.Span;
            Span<byte> decoderDest = decoderDestPoisoned.Span;

```